utilities/checkpoint/checkpoint.cc

```diff
@@ -62,6 +62,8 @@ Status CheckpointImpl::CreateCheckpoint(const std::string& checkpoint_dir) {
   Status s;
   std::vector<std::string> live_files;
   uint64_t manifest_file_size = 0;
+  bool allow_2pc = db_->GetDBOptions().allow_2pc;
+  uint64_t min_log_number_to_keep = port::kMaxUint64;
```

Comment:
Can we also fix unity? It's complaining that this variable shadow's `min_log_number_to_keep` in `db/internal_stats.cc`